ext/acorn-android/src/test/java/com/nhaarman/acorn/android/dispatching/internal/DefaultActivityHandlerTest.kt

```diff
@@ -51,7 +51,7 @@ internal class DefaultActivityHandlerTest {
         )
 
         @Test
-        fun `withScene starts Intent`() {
+         fun `withScene starts Intent`() {
```

Comment:
**[ktlint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
Unexpected indentation (9) (it should be 12) (cannot be auto-corrected)